tests/functional/cephadm/group_vars/all

```diff
@@ -5,4 +5,5 @@ cluster_network: "192.168.31.0/24"
 dashboard_admin_password: $sX!cD$rYU6qR^B!
 ceph_docker_registry: quay.ceph.io
 ceph_docker_image: ceph-ci/daemon-base
-ceph_docker_image_tag: latest-master-devel
\ No newline at end of file
+ceph_docker_image_tag: latest-master-devel
+containerized_deployment: true
```

Comment:
What was the issue with this ?

As far as I remember, the cephadm playbook doesn't use that variable nor the roles included in there